packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -16,8 +20,23 @@ const AvatarContainer = styled.div`
   float: left;
 `;
 
-const OpenQueueCard = ({ queue }: OpenQueueCard) => {
+const EditOutlinedFloatedRight = styled(EditOutlined)`
+  float: right;
+`;
+
+const OpenQueueCard = ({ queue, isTA }: OpenQueueCard) => {
+  const [editingNotes, setEditingNotes] = useState(false);
   const staffList = queue.staffList;
+
+  const updateNotes = (value: any) => {
+    const notes = (value.target as any).value;
+    if (notes) {
```

Comment:
this should probably check if `notes !== queue.notes`

Reply:
talked about this earlier with dajinder